### ISSUE TITLE
Public Sharable Link for Albums

### ISSUE THREAD

**Issue by AARtwo** [LibrePhotos/librephotos#220]
Currently you can publicly share individual photos to people with a URL. It doesn't appear possible to share albums in the same way.

This is currently possible with Google photos, I, and I'm sure many others are using this as a replacement for this service. I like to share photos from events with family/friend and don't want to force them to make an account to see it.

**Comment by Seneliux** [comment]
Or make Public photos very public :)
Is it possible to create an guest account, then copy login url?
I tried https://LIBREPHOTOS/login?username=guest&password=guest_password, but does not worked.
May it is possible without filling user/password form to login directly?

One shared album = one link.

**Comment by derneuere** [comment]
You can see all public photos without being logged in under /user/username

**Comment by wjbeckett** [comment]
I'd like to vote for this feature too.
The public photos URL is fine for when you want to share individual photos or a few photos. But I'd like to create an album, and then make that album public. For example, I have photos of my brothers wedding and I want to share those photos with family members, but only those photos. Perfect use case here would be to create an album of those photos, and then share that album only. I currently do that with Photoprism, but I would like to move away from Photoprism to LibrePhotos and this is one thing I'm missing that is quite important to me.

**Comment by amido-selstam** [comment]
Yes, this is right now a kind of dealbreaker for me. I take a lot of photos of sport events and want to share it with respective players in teams and I usually pass them a link they can use to get access to the complete album - no registration or need for an account. This is what Google Photos Share excels at - and I would happily use LibrePhotos more "publically" if this was implemented.

**Comment by selstam** [comment]
Yes, this is right now a kind of dealbreaker for me. I take a lot of photos of sport events and want to share it with respective players in teams and I usually pass them a link they can use to get access to the complete album - no registration or need for an account. This is what Google Photos Share excels at - and I would happily use LibrePhotos more "publically" if this was implemented.

**Comment by OCT0PUSCRIME** [comment]
Weighing in with my desire for this feature as well.

**Comment by teaboyteaboy** [comment]
> 

The /user/username doesn't seem to show all public images, at least not if you're not logged in.

**Comment by Frankenberrypi** [comment]
> > 
> 
> The /user/username doesn't seem to show all public images, at least not if you're not logged in.

This seems to be true for my installation also.

**Comment by Gremious** [comment]
I would also like to add my 2 cents that hidden images should continue be visible in albums.

Usecase is simple - I want to hide more-personal images from the regular public view, while being able to share them in an album to my partner. So far, literally no other photo gallery app I've tried allows me to do that. Immich comes closest by allowing me to have 2 separate accounts for free (what I had with google photos).

LibrePhotos having a separation between private and hidden is already a blessing. Now I only need to be able to share said hidden photos as albums.

**Comment by CodingGhost** [comment]
I would also love to have that feature!

**Comment by yawnbox** [comment]
over 4 years? please, please make it easier to share albums publicly

**Comment by derneuere** [comment]
It is now available on dev and will be in the next latest release :)
